client/src/store/new-request-module.ts

```diff
@@ -2370,32 +2372,44 @@ export class NewRequestModule extends VuexModule {
       } else {
         await errorModule.setAppError({ id: 'complete-payment-error', error: error.message } as ErrorI)
       }
+      return null
     }
   }
   @Action
-  async rollbackNameRequest ({ nrId, action }): Promise<any> {
+  async rollbackNameRequest (nrId: number, action: string): Promise<boolean> {
     try {
-      const validRollbackActions = [
-        ROLLBACK_ACTIONS.CANCEL
-      ]
+      // only cancel action is supported atm
+      const validRollbackActions = [ROLLBACK_ACTIONS.CANCEL]
+
+      // safety checks
+      if (!nrId) {
+        console.log('rollbackNameRequest(), invalid NR id') // eslint-disable-line no-console
+        return false
```

Comment:
This will alert us (while we test) of problems, but should let the code proceed in Prod.

Reply:
NB: logs were changed to errors (so we will see Prod errors), though this should never happen (haha).